bills-app/src/main/webapp/script.js

```diff
@@ -767,10 +767,12 @@ const mockResponse = {
     }]
 }
 
-// Fetched the upcoming bills from the api
-const getUpcomingBills = () => {
+
+const getAPIStatus() = () => {
```

Comment:
It's a very small detail, but if this function gets the upcoming bills, wouldn't it be better described with the name `getUpcomingBills` ?